src/frontend/sharedComponents/ThumbnailScrollView.tsx

```diff
@@ -58,10 +64,9 @@ export const Thumbnail = ({photo, style, size, onPress}: ThumbnailProps) => {
   );
 };
 
-export const ThumbnailScrollView = () => {
+export const ThumbnailScrollView = (props: {photos: (Photo | undefined)[]}) => {
```

Comment:
It feels a little weird that we'd allow an array with `undefined` in it. What's the use case for that?

Reply:
it's so we can keep placeholders for still loading images